cmd/config-downloader/downloader.go

```diff
@@ -154,11 +157,12 @@ func main() {
 	if region == "" && downloadLocation != locationDefault {
 		fmt.Println("Unable to determine aws-region.")
 		if mode == config.ModeEC2 {
-			errorMessage = fmt.Sprintf("Please check if you can access the metadata service. For exampe, on linux, run 'wget -q -O - http://169.254.169.254/latest/meta-data/instance-id && echo' ")
+			errorMessage = fmt.Sprintf("E! Please check if you can access the metadata service. For example, on linux, run 'wget -q -O - http://169.254.169.254/latest/meta-data/instance-id && echo' ")
```

Comment:
I don't think that the downloader executable points to the telegraf logger in any way, so I don't think that the `E!` notation helps much outside of letting us know semantically that this is an "error". Just calling that out. It's not a breaking change so I don't mind keeping it.

Reply:
Agree on this. Just my habit of grouping things to error.